plugins/module_utils/network/eos/config/ospfv3/ospfv3.py

```diff
@@ -151,6 +151,13 @@ def _global_compare(self, want, have):
                     self._module.warn(
                         "'timers throttle' command is deprecated by 'timers lsa / timers spf'"
                     )
+                    entry.pop("throttle")
+                    continue
+                if entry.get("lsa") and not isinstance(entry["lsa"], dict):
+                    self._module.warn(
+                        "timers.lsa accepts only dict. Please refer the documentaion."
```

Comment:
Is it at all possible for a facsimile of the previous functionality to be preserved here? Because otherwise I don't think there's a point to mangling the argspec so we can ignore an option.